index.js

```diff
@@ -0,0 +1,107 @@
+var stateData = JSON.parse(JSON.stringify(require('./state.json')));
+
+// Load external file with states and associated tribes
+var stateTribeMappings = require('./state_mappings.js');
+
+// Get all states that are associated with tribal names
+var statesFromTribes = Object.keys(stateTribeMappings.mappings());
+
+// Create two separate FeatureCollections
```

Comment:
This is great! You taught me something new. I typically would have pushed the tribal names to two different arrays and then have used [turf-featurecollection](https://github.com/turf-junkyard/turf-featurecollection), but your way is cleaner with less dependencies :tada: